var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -606,3 +606,10 @@ def build_amd(self):
     def install_test(self):
         with working_dir("test"):
             python("run_test.py")
+
+    @property
+    def cmake_prefix_paths(self):
+        cmake_prefix_paths = [
+            join_path(self.prefix, self.spec["python"].package.platlib, "torch/share/cmake")
```

Comment:
Does this work?
```suggestion
            join_path(python_platlib, "torch", "share", "cmake")
```

Reply:
It didn't complain yet, so syntax passes. I assume this will work indeed.